epictrack-api/src/api/services/work_phase.py

```diff
@@ -150,6 +150,7 @@ def _find_work_phase_status(cls, work_id, work_phase_id, work_phases):
         events = EventService.find_events(work_id, event_categories=PRIMARY_CATEGORIES)
         if work_phase_id is not None:
             work_phases = [wp for wp in work_phases if wp.id == work_phase_id]
+        index  = 1
         for work_phase in work_phases:
```

Comment:
This could be replaced with 
```
for index, work_phase in enumerate(work_phases, start=1):
    ...
```

And avoid manually initializing and updating index.